### ISSUE TITLE
Pretrained weights

### ISSUE THREAD

**Issue by colindecourt** [Zhuanglong2/T-RODNet#10]
Hi, 

I want to evaluate your model on the test set of the ROD2021 dataset. I trained your model using the configuration file and the training/validation set you provided. I trained the model for 60 epochs and used the final model (after epoch 60) to make the inference on the ROD2021 test set. 
After uploading the results on the [evaluation platform ](https://codalab.lisn.upsaclay.fr/competitions/1063#), I got a mAP of 69.90. However, there is no indication of the weights you use for your inference. For a fair evaluation, could you provide the pre-trained weights you used for the results of your paper, please? 

Kind regards,
Colin

**Comment by Zhuanglong2** [comment]
We have resolved the issue where T-RODNet training fails on some graphics cards. Please download our latest project for training.
